Make-a-pull-request.md

```diff
@@ -106,9 +106,9 @@ Once your feature is ready, you can open a pull request (PR)!
 
 * Click "Create pull request". If applicable, verify that the issue your PR fixes is linked in the "Development" section on the right. (If not, update your description to clearly reference the issue, using the format "Fix #bugnum".)
 
-* Oppiabot will check that you filled out the PR description correctly. If you didn't, it will leave a comment explaining what you need to do to fix it. More details on the kinds of comments Oppiabot leaves are coming soon.
+* Before getting a review, **double-check** that the changes in the "Files Changed" tab are exactly what you want them to be, and that there are no extra changes. (If there are any issues, make a new commit to fix them and push your changes.)
```

Comment:
We already ask contributors to check their changes twice: in step 3 using `git diff upstream/develop` before pushing their commits and at the beginning of step 4 before clicking "create pull request". Do we really need them to check a third time? If the concern is that contributors aren't actually checking their changes, maybe make the instruction at the top of step 4 more prominent?

Reply:
We've seen instances of superfluous changes during code reviews. It would be helpful to emphasize this point once more. Additionally, ensuring there are no unnecessary changes is part of the self-review process. So I think it is fine to mention it here after a PR is opened. wydt?

Reply:
Done. (As a case in point, I _had_ missed the instruction at the top of step 4 :P )